packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -354,3 +331,83 @@ class _DomainCheckTable extends StatelessWidget {
     );
   }
 }
+
+class _DeepLinkListViewTopPanel extends StatefulWidget {
```

Comment:
this can be a stateless widget. To access the controller object you can access it directly in the build method from provider: `Provider.of<DeepLinksController>(context)`, or you can either pass it in as a parameter from the parent